.travis.yml

```diff
@@ -4,17 +4,12 @@ dist: bionic
 php:
   - 7.3
 
-env:
-  matrix:
-    - COMPOSER_FLAGS="--prefer-lowest"
-    - COMPOSER_FLAGS=""
-
 before_install:
   - curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
   - chmod +x ./cc-test-reporter
 
 before_script:
-  - travis_retry composer update ${COMPOSER_FLAGS} --no-interaction --prefer-source
+  - travis_retry composer install --no-interaction
```

Comment:
I prefer not to change this. I have encountered issues affecting the lowest package versions and this way I can ensure that at least I'm supporting the base versions of my dependencies.